app/rest/docker/container.js

```diff
@@ -40,6 +40,10 @@ angular.module('portainer.rest')
     exec: {
       method: 'POST', params: {id: '@id', action: 'exec'},
       transformResponse: genericHandler
+    },
+    inspect: {
+      method: 'GET', params: { id: '@id', size: '@size', action: 'json' },
```

Comment:
The `size` parameter is not used here, you can remove it. You can also remove the `timeout` value, as we don't use any timeout in other requests.